changelogs/fragments/1246-flatpak-use-non-interactive-argument.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- flatpak - add the use of the --non-interactive argument instead of -y when possible.
```

Comment:
```suggestion
- flatpak - use of the C(--non-interactive) argument instead of C(-y) when possible (https://github.com/ansible-collections/community.general/pull/1246).
```

Reply:
I'm sorry but I don't understand the use of the `C()`. I tried to do some research about it but I wasn't successful. Could you link me a doc or explain it briefly if it's not too much trouble?

Reply:
@TGuimbert, hi
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation
I'll highlight things a bit

Reply:
Sorry, looks like I was already partially asleep ;) In changelog fragments, `C(...)` does not work. They are reStructuredText, so they need double backticks. I've adjusted my suggestion.

Reply:
@Andersson007 @felixfontein Thanks for the clarification!